examples/flutter_gallery/lib/demo/material/banner_demo.dart

```diff
@@ -0,0 +1,117 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import '../../gallery/demo.dart';
+
+enum BannerDemoAction {
+  reset,
+  showMultipleActions,
+  showLeading,
+}
+
+class BannerDemo extends StatefulWidget {
+  const BannerDemo({ Key key }) : super(key: key);
+
+  static const String routeName = '/material/banner';
+
+  @override
+  _BannerDemoState createState() => _BannerDemoState();
+}
+
+class _BannerDemoState extends State<BannerDemo> {
+  bool displayBanner = true;
```

Comment:
These can all be private. Also, there is no reason to assign these here since initBanner does that in initState

Reply:
Agreed to making these private, although I think you should leave assignment here at declaration.

https://dart.dev/guides/language/effective-dart/usage#do-initialize-fields-at-their-declaration-when-possible

Reply:
done